src/sentry/relay/config/__init__.py

```diff
@@ -170,9 +170,41 @@ def get_filter_settings(project: Project) -> Mapping[str, Any]:
     if csp_disallowed_sources:
         filter_settings["csp"] = {"disallowedSources": csp_disallowed_sources}
 
+    try:
+        generic_filters = _get_generic_project_filters()
+    except Exception:
+        logger.exception(
+            "Exception while building Relay project config: error building generic filters"
+        )
+    else:
+        if generic_filters and len(generic_filters["filters"]) > 0:
+            filter_settings["generic"] = generic_filters
+
     return filter_settings
 
 
+class GenericFilter(TypedDict):
+    id: str
+    isEnabled: bool
+    condition: Mapping[str, str] | None
```

Comment:
Can we type this out? There is already types for this in https://github.com/getsentry/sentry/blob/38a91b6dc20655fa299a6b2a13b10231d9480c39/src/sentry/dynamic_sampling/rules/utils.py#L95-L110 (might need to be extended).